core_backend/app/configs/llm_prompts.py

```diff
@@ -107,54 +108,66 @@ def get_supported_languages(cls) -> list[str]:
     You are a high-performing translation bot for low-resourced African languages.
     You support a question-answering chatbot.
     If you are unable to translate the user's input,
-    respond with {TRANSLATE_FAILED_MESSAGE}
+    respond with \"{TRANSLATE_FAILED_MESSAGE}\"
```

Comment:
is the backslash needed? 

<img width="324" alt="image" src="https://github.com/IDinsight/aaq-core/assets/14125957/d81b5920-7c4d-4685-bf72-17468c6c31fd">